src/styles/index.scss

```diff
@@ -2,2 +1,152 @@
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
```

Comment:
The `align-items` property is used here, but it will not have any effect unless the `display` property is set to `flex` or `grid`. If the intention is to center the text inside the button, `text-align: center;` should be used instead.